File: Assesment Folder/DARSHAN_RAJ_ASSESSMENT_11/Question2/Actions/LoginAction.py
from Pages.LoginPage import LoginPage

class LoginAction:

    def __init__(self,driver):
        self.driver = driver
    
    def open_login(self):
        self.click(self.my_account)
        self.click(self.login)
    
    def login(self,user,passw):
        self.enter_text(LoginPage.email,user)
        self.enter_text(LoginPage.password,passw)
        self.click(LoginPage.login_btn)
    def get_error(self):
        return self.get_text(LoginPage.warning)
    
    

   

File: Assesment Folder/DARSHAN_RAJ_ASSESSMENT_08/Assessment/src/test/java/com/actions/LoginAction.java
package com.actions;

import org.openqa.selenium.WebDriver;

import com.pages.LoginPage;

public class LoginAction {

	LoginPage page;

	public LoginAction(WebDriver driver) {

		page = new LoginPage(driver);
	}

	public void login(String email, String password) {

		page.email.sendKeys(email);

		page.password.sendKeys(password);

		page.loginButton.click();
	}
}
